docker-compose.yml

```diff
@@ -67,5 +69,14 @@ services:
     networks:
       - zodiac-bots-network
 
+  smtp:
+    image: maildev/maildev
+    environment:
+      - MAILDEV_WEBMAIL_USER=kleros
+      - MAILDEV_WEBMAIL_PASSWORD=sorelk
```

Comment:
This is fixed?

Reply:
@wadader sorry, what is wrong with it? I have the webmail configured but not exposed. When I want to access the webmail, I expose the ports by loading my `docker-compose.local.yml` file which looks like:

```yml
version: "3"

services:
  db:
    ports:
      - "127.0.0.1:15432:5432"

  smtp:
    ports:
      - "127.0.0.1:1080:1080"
```

Reply:
Alright, I see now